docs/tutorials/01_ui/viz_ui_slider.py

```diff
@@ -81,14 +116,25 @@ def translate_cube(slider):
 
 show_manager.scene.add(cube)
 show_manager.scene.add(ring_slider)
-show_manager.scene.add(line_slider)
+show_manager.scene.add(hor_line_slider_text_top)
+show_manager.scene.add(hor_line_slider_text_bottom)
+show_manager.scene.add(ver_line_slider_text_left)
+show_manager.scene.add(ver_line_slider_text_right)
+show_manager.scene.add(hor_line_slider)
+show_manager.scene.add(ver_line_slider)
+
 
 ###############################################################################
 # Visibility by default is True
 
 cube.SetVisibility(True)
 ring_slider.set_visibility(True)
-line_slider.set_visibility(True)
+hor_line_slider_text_top.set_visibility(True)
+hor_line_slider_text_bottom.set_visibility(True)
+ver_line_slider_text_left.set_visibility(True)
+ver_line_slider_text_right.set_visibility(True)
+hor_line_slider.set_visibility(True)
+ver_line_slider.set_visibility(True)
```

Comment:
same, you can remove `hor_line_slider` and `ver_line_slider`

Reply:
> Ok, the last review, I just tested it and it works well. After your last update, I will merge your PR.
> 
> Any other comments on this PR?

Got it, please do let me know if any further changes needed.
cheers.